.github/workflows-source/ci-main.yml

```diff
@@ -235,6 +253,39 @@ jobs:
           # Clean up the output base for the next run
           sudo rm -rf /var/tmp/bazel-output
 
+  pocketic-on-windows:
+    name: PocketIC tests on Windows
+    needs: [bazel-test-all]
+    <<: *pocketic-windows-trigger
+    <<: *windows-setup
+    steps:
+      - <<: *checkout
+      - name: Download PocketIC server
+        uses: actions/download-artifact@v4
+        with:
+          name: pocket-ic-server
+      - name: Setup WSLv2
+        uses: vedantmgoyal9/setup-wsl2@main
+      - name: Code linting
+        run: |
+          cargo clippy --locked --all-features -p pocket-ic -p pocket-ic-test-canister --all-targets -- -D warnings -D clippy::all -D clippy::mem_forget -C debug-assertions=off
+      - name: Build the test canister
+        run: |
+          cargo build --release --locked -p pocket-ic-test-canister --target wasm32-unknown-unknown
+      - name: Shrink the test canister
+        run: |
+          curl -L https://github.com/dfinity/ic-wasm/releases/download/0.9.3/ic-wasm-linux64 -o ic-wasm
+          chmod +x ic-wasm
+          ./ic-wasm -o test-canister.wasm target/wasm32-unknown-unknown/release/pocket-ic-test-canister.wasm shrink
+        shell: wsl-run {0}
```

Comment:
Alternatively, we could fetch the test canister from bazel-test-all by uploading/downloading its artifact.

Reply:
I would prefer that option since then we don't have to update this URL all the time.